monitors/filterWords.js

```diff
@@ -0,0 +1,32 @@
+exports.conf = {
+  enabled: true,
+  // Change to true if the filter should ignore other bots
+  ignoreBots: false,
+  // Change to true if the filter should ignore selfBots
+  ignoreSelf: false,
+};
+
+exports.run = (client, msg) => {
+  // Do not delete messages from the Bot itself otherwise it will run on a loop when it sends msgs in a DM to author or reports to a channel.
+  if (msg.author.bot) return;
+  // Add your list of words you want to filter. This is a very powerful filter so be careful of using. For example, the word 'assign' will not be allowed if the word `ass` is present in the swear words. In other filters, they are easily breakable by typing 'ass.' but this filter will delete these types of breaks.
+  const swearWords = ["damn", "shucks", "frak", "ass"];
+
+  const badWords = swearWords.filter(word => msg.content.toLowerCase().includes(word));
+  if (badWords.length > 0) {
+    let badWordList = [];
+    for (let i = 0; i < badWords.length; i++) {
+      badWordList += badWords[i];
+    }
+    // DM the user regarding what words caused the message to be deleted.
+    msg.author.send(`I deleted the message below because you used a word that is not allowed. : **${badWordList}**`);
+    // DM the user with the message incase they wish to copy paste -> edit and resend. Imagine what happens when by accident a user gets the message deleted for a mistake.
+    msg.author.send(`${msg.content}`);
+    // Delete a message
+    msg.delete()
```

Comment:
You should check if the bot has permissions to delete messages (msg.deletable) before processing everything.